Automattic/jetpack
# Search: Customize links should point to Customberg

Related to #29374.

## Proposed changes:

Changes links for customizing search from the Search section of Jetpack Cloud to Customberg, the search customizer interface.

## Other information:


## Jetpack product discussion


Part of the Jetpack Excellence effort p1HpG7-kKy-p2.

## Does this pull request change what data or activity we track or use?


No.

## Testing instructions:






* Spin up a Jurassic Ninja site with this branch enabled on the Jetpack plugin.
* Go to Search → Start for free → Complete free purchase.
* Go to Jetpack → Recommendations → Click Customize search CTA.
* Ensure that you're taken to Customberg.
